src/utils/get-video-from-ans/index.js

```diff
@@ -0,0 +1,12 @@
+const embedSource = (content) =>
+	({
+		video: content,
+		story: content?.promo_items?.lead_art,
```

Comment:
What about `content.promo_items.basic` - a video can be here too

Reply:
hmm, the `engine-theme-sdk` did not account for actually extracting that.  It looks like the data object has it, but it was never accessed.
![image](https://user-images.githubusercontent.com/2287238/179843945-d0394074-1ea3-4cae-a435-92d370c0d7c8.png)

Reply:
I'm also not seeing anything in our code that would access `promo_items.basic.embed_html`

Reply:
The three blocks (`extra-large-promo`, `large-promo`, and `top-table-list`) that get the `embed_html` only get it from the base `content` or the `lead_art`
https://github.com/WPMedia/arc-themes-blocks/blob/bcadf86b25cac99088e88bd3e0a89f7cc4a551c2/blocks/extra-large-promo-block/features/extra-large-promo/default.jsx#L77
https://github.com/WPMedia/arc-themes-blocks/blob/bcadf86b25cac99088e88bd3e0a89f7cc4a551c2/blocks/large-promo-block/features/large-promo/default.jsx#L75
https://github.com/WPMedia/arc-themes-blocks/blob/bcadf86b25cac99088e88bd3e0a89f7cc4a551c2/blocks/top-table-list-block/features/top-table-list/default.jsx#L148

Reply:
Ok - we can cross that bridge later then, if none of our current blocks need it